doc/api/users.md

```diff
@@ -58,6 +58,7 @@ GET /users
     "is_admin": false,
     "avatar_url": "http://localhost:3000/uploads/user/avatar/1/cd8.jpeg",
     "can_create_group": true
```

Comment:
There's a comma missing at the end of the line. Note the red colon on the next line.